packages/autocomplete-core/src/createAutocomplete.ts

```diff
@@ -62,13 +62,17 @@ export function createAutocomplete<
   props.plugins.forEach((plugin) =>
     plugin.subscribe?.({
       ...setters,
+      props,
```

Comment:
Looks like we only need `navigate` from the props.

Reply:
I'm going to mark this comment resolved since your [other comment](https://github.com/algolia/autocomplete/pull/1082#discussion_r1097195203) touches on this as well and would invalidate this if we go that route. Feel free to mark unresolved if you disagree. 😄